controllers/post.js

```diff
@@ -1,11 +1,16 @@
 const Post = require("../models/Post");
 
 exports.findAll = (req, res) => {
-  let filter = {};
+  let filter,
+    sort = {};
```

Comment:
Are you certain that declaration makes filter = {}

Reply:
it doesn't, fixed in new commit